### PR TITLE
Incident

### PR DESCRIPTION
**Pull request by daniellee0**

### CODE REVIEW

File: server/routes.js [hunk 1]
```diff
@@ -501,4 +506,59 @@ app.route("/villages")
...
+	// Add new incident report
+	.post((req, res) => {
+		res.setHeader('Access-Control-Allow-Origin', 'http://localhost:3000');
+		let incident = req.body;
+		let newIncidentID = "";		
+		incident.injury = (incident.injury == 'true');
+		incident.emergencyRoom = (incident.emergencyRoom == 'true');
+		incident.policeReport = (incident.policeReport == 'true');
+		res.setHeader('Access-Control-Allow-Origin', 'http://localhost:3000');
+		con.query('INSERT INTO INCIDENTS (INCIDENT_DATE, TIME, VID, LOCATION, DESCRIPTION, INJURY, INJURY_DESCRIPTION, ER_VISIT, ER_HOSPITAl, POLICE_REPORT, PR_NUMBER, AUTHOR_ID , REVIEWER_ID , AUTHOR_DATE) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)',
```
Comment (kushaljh) [inline server/routes.js]: typo: ER_HOSPITAl

File: server/routes.js [hunk 2]
```diff
@@ -501,4 +506,59 @@ app.route("/villages")
...
 	})
 
+
+
+
+
+app.route("/incidentReport")
+	// Add new incident report
+	.post((req, res) => {
+		res.setHeader('Access-Control-Allow-Origin', 'http://localhost:3000');
```
Comment (kushaljh) [inline server/routes.js]: use the domain from commons.js and not hardcoded values here

File: server/routes.js [hunk 3]
```diff
@@ -501,4 +506,59 @@ app.route("/villages")
...
+		let incident = req.body;
+		let newIncidentID = "";		
+		incident.injury = (incident.injury == 'true');
+		incident.emergencyRoom = (incident.emergencyRoom == 'true');
+		incident.policeReport = (incident.policeReport == 'true');
+		res.setHeader('Access-Control-Allow-Origin', 'http://localhost:3000');
```
Comment (kushaljh) [inline server/routes.js]: Why do you set this header again?

File: src/ResidentDirectory.js [hunk 4]
```diff
@@ -118,7 +130,6 @@ class ResidentDirectory extends Component {
 
                     {this.state.displayCard &&
                     <Card style={{width: '30rem', marginTop: "30px", marginLeft: "500px"}}>
-                        <Card.Img variant="top" src="placeHolder.png"/>
```
Comment (kushaljh) [inline src/ResidentDirectory.js:121]: Why was this removed?

**Review by kushaljh — APPROVED** [review]

**Review by wilsmic2 — APPROVED** [review]
